### PR TITLE
Migrate some sig-net jobs to podutils

### PR DESCRIPTION
**Pull request by rikatz**
Fixes: #20324 (partially, periodic jobs still need to be migrated)

This PR migrates some Sig Network Prow jobs to use podutils 

In a future, we will also need to migrate to kubetest2, and check the periodic tests (figuring out owners, if this is still something, etc).

/cc @aojea @spiffxp

### CODE REVIEW

**Comment by rikatz** [comment]
/cc @krzyzacy please take a look into DNS test (it seems you're the original author and the test wasn't touched since that time)

Thanks!

**Comment by aojea** [comment]
let´s go with ipvs and kind jobs first, that are the ones I have under control, and once we move them we ping dns tests maintainers
does it sounds good?

**Comment by aojea** [comment]
> let´s go with ipvs and kind jobs first, that are the ones I have under control, and once we move them we ping dns tests maintainers to migrate those other jobs
> does it sounds good?

unless @amwat, @MrHohn @prameshj are ok with start migrating them, of course :smile:

**Review by amwat — COMMENTED** [review]
Thanks a lot for tackling this!
The pure conversion parts of bootstrap -> podutils LGTM (also matches the other jobs here)

/lgtm

File: config/jobs/kubernetes/sig-network/sig-network-misc.yaml [hunk 1]
```diff
@@ -107,20 +110,18 @@ presubmits:
...
+        command:
+        - runner.sh
+        - /workspace/scenarios/execute.py
         args:
-        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
-        - --root=/go/src
-        - --upload=gs://kubernetes-jenkins/pr-logs
-        - --scenario=execute
-        - --
         - "true"
```
[the remark this answers is not in the document]
Comment (rikatz) [reply config/jobs/kubernetes/sig-network/sig-network-misc.yaml:125]: Yeah, sorry folks, was below a low of things here. I'll review this until wednesday. Will fix the branch (sorry @aojea for the other failing job :P ) and the return true here

**Comment by aojea** [comment]
> Yeah, sorry folks, was below a low of things here. I'll review this until wednesday. Will fix the branch (sorry @aojea for the other failing job :P ) and the return true here

no sorry at all, this things happens all days and only fails who tried.
If you don't mind, l´ll put this on hold until you can follow up on the comments
/hold

**Comment by spiffxp** [comment]
I'll try to look at this next week if you get a chance to rebase

**Comment by rikatz** [comment]
Sure! I was prioritizing some other PRs review before code freeze, will take a look at this ASAP!

**Pull request closed without merging**